practice-app/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:10
+
+# Create app directory
+WORKDIR .
+
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+# where available (npm@5+)
+COPY backend/package*.json ./
+
+RUN npm install
+# If you are building your code for production
+# RUN npm ci --only=production
+
+# Bundle app source
+COPY . .
+
+EXPOSE 9876
+CMD [ "node", "backend/app.js" ]
```

Comment:
I propose `RUN npm start` would be more proper @egirgin

Reply:
solved [here](https://github.com/bounswe/bounswe2020group4/pull/102/commits/96d16ebf00d949904727fc7ddf85452a6144c703)